pkgs/development/python-modules/vulkan/vulkanapi-paths.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/vulkan/_vulkan.py b/vulkan/_vulkan.py
+index 4a2c542..ed01f69 100644
+--- a/vulkan/_vulkan.py
++++ b/vulkan/_vulkan.py
+@@ -100,7 +100,7 @@ _cast_ptr = _cast_ptr3 if PY3 else _cast_ptr2
+ 
+ 
+ # Load SDK
+-_lib_names = ('libvulkan.so.1', 'vulkan-1.dll', 'libvulkan.dylib')
++_lib_names = ('@vulkan_loader@/lib/libvulkan.so.1', 'vulkan-1.dll', 'libvulkan.dylib')
```

Comment:
My solution is probably better as this does not address compatibility issues it may have on macOS

Reply:
Maybe. Ideally there'd be a way to test this stuff on darwin. I might have a poke at setting up a hackintosh to test.